autogluon/requirements.txt

```diff
@@ -1,2 +1,7 @@
- pandas==1.3.3
- autogluon==0.3.1
+--find-links https://download.pytorch.org/whl/torch_stable.html
+torch==1.9.1+cpu
+torchvision==0.10.1+cpu
```

Comment:
CPU-only version for torch - 185Mb vs 850+ for regular

Reply:
this is okay. thanks